production/helm/fluent-bit/values.yaml

```diff
@@ -10,6 +10,7 @@ config:
   port: 2020
   loglevel: warn
   lineFormat: json
+  k8sLoggingParser: "Off"
```

Comment:
Sorry but this field is missing in the doc.

Reply:
sorry, I have just added it.